SickDesignSystem/synergy
# fix: 🐛 Knobs use type "numeric" when it should use "select" in Storybook

# Pull Request

## 📖 Description

This PR upgrades to the latest version of `@storybook/web-components`, which fixes the knob types.

### 🎫 Issues

Closes #82.

## 📑 Test Plan

- Run `pnpm i -r` in the root of this branch.
- Make sure to create a new bundle by using `pnpm build`.
- Move to `packages/docs` and issue `pnpm start`.
- Have a look at the button stories `variant` knob. It should now be adjustable and displayed as a list of combo-boxes.

## ✅ Checklist
